app/controllers/questions_controller.rb

```diff
@@ -9,12 +9,24 @@ class QuestionsController < ApplicationController
 
   def index
     @questions = Question.order("id DESC")
-
     respond_to do |format|
       format.json { render json: @questions }
     end
   end
 
+  def count
```

Comment:
What is this and the subset method/routes for? You should be able to put the limit/offset in the index method, no?

Reply:
Yes we could definitely override index. Sometimes I make new routes to ensure it doesn't break the old and to show comparison. In theory we wouldn't need anything but the paginated version, but wasn't sure if we had a use case for the index with all questions.

Reply:
See updated pull request.